src/network/networkd-link.c

```diff
@@ -2163,6 +2164,17 @@ static int link_set_can(Link *link) {
                         return log_link_error_errno(link, r, "Could not append IFLA_CAN_RESTART_MS attribute: %m");
         }
 
+        cm.mask = CAN_CTRLMODE_3_SAMPLES;
+
+        if (link->network->can_triple_sampling)
```

Comment:
Not sure, but we usually store such flags in tristate: `-1` is not unset, `0` is disabled, `1` is enabled. And only if the flag has an non-negative value, then send relevant message.